assets/css/template/_package-view.scss

```diff
@@ -108,6 +108,21 @@
   }
 }
 
+.audit-logs-list {
+  padding: 0;
+  list-style: none;
+
+  li {
```

Comment:
This block is copy-pasted from L91, do we need to make them a mixin or something?